drivers/hmis/app/graphql/types/hmis_schema/client.rb

```diff
@@ -253,6 +254,14 @@ def addresses
       load_ar_association(object, :addresses)
     end
 
+    def hud_chronic
+      return unless current_permission?(permission: :can_view_hud_chronic_status, entity: object)
+
+      # client.hud_chronic causes n+1 queries
```

Comment:
Is that because it is using `GrdaWarehouse::Hud::Enrollment`? instead of `Hmis::Hud::Enrollment`?  We could probably re-work that.  We should be able to preload everything if we're ever calling it on more than one client/enrollment at a time.

Reply:
I just wanted to call out that we shouldn't call this on a collection of clients. It seems like we don't have that need right now so no action is needed.